site/content/en/docs/reference/plugins/consumer_restriction.md

```diff
@@ -33,7 +33,7 @@ Only one of `allow` or `deny` can be configured.
 | Name | Type   | Required | Validation   | Description          |
 |------|--------|----------|--------------|----------------------|
 | name | string | True     | min_len: 1   | Name of the Consumer |
-
+| methods | string[] | False   |  | List of HTTP methods allowed/prohibited for Consumer |
```

Comment:
Would you add the pattern to "Validation" before I can merge it? Thanks!

Reply:
> Would you add the pattern to "Validation" before I can merge it? Thanks!

The validation is annotated with 'must be uppercase'